.github/workflows/android-ci-pull.yml

```diff
@@ -59,6 +59,14 @@ jobs:
           submodules: recursive
           fetch-depth: 0
 
+      - name: Setup Android SDK
+        uses: android-actions/setup-android@v2
+
+      - name: Setup cmake
```

Comment:
because its the one required by the android docker container from platforms/android/docker that it all runs on
https://github.com/maplibre/maplibre-gl-native/actions/runs/3142446133/jobs/5106047299